auth0_flutter/test/web/auth0_fluter_web_test.dart

```diff
@@ -97,6 +97,21 @@ void main() {
     expect(params.max_age, 10);
   });
 
+  test('loginWithRedirect supports custom parameters', () async {
+    when(mockClientProxy.isAuthenticated())
+        .thenAnswer((final _) => Future.value(false));
+
+    await auth0.loginWithRedirect(parameters: {'screen_hint': 'signup'});
+
+    final params = verify(mockClientProxy.loginWithRedirect(captureAny))
+        .captured
+        .first
+        .authorizationParams;
+
+    expect(params != null, true);
```

Comment:
Could this be `expect(params, notNull);`?

Reply:
Certainly can, fixed in [15142e2](https://github.com/auth0/auth0-flutter/pull/226/commits/15142e277341bfedc4e44d7d7cd5b5ec41cb62a2)